packages/datetime/src/common/dateUtils.ts

```diff
@@ -161,3 +161,18 @@ export function fromMomentToDate(momentDate: moment.Moment) {
         );
     }
 }
+
+/**
+ * Translate a DateRange into a two-element array of moments, adjusting the
+ * local timezone into the moment one (a no-op unless moment-timezone's
+ * setDefault has been called).
+ */
+export function fromDateRangeToMomentArray(dateRange: DateRange) {
```

Comment:
does it make sense to type this as `[moment.Moment, moment.Moment]` (a two-element tuple), like `DateRange` is `[Date, Date]`?